deploy/olm-catalog/service-assurance-operator/0.1.0/infra.watch_serviceassurances_crd.yaml

```diff
@@ -20,62 +20,18 @@ spec:
   validation:
     openAPIV3Schema:
       properties:
-        apiVersion:
```

Comment:
This is stripped probably because you generated with olm-catalog 0.13.0. I'd suggest going back a version so that this isn't stripped out.